README.md

```diff
@@ -1,5 +1,13 @@
 # Obsidian Filename Linter Plugin
 
+## Description
+
+Lint filenames for invalid or troublesome characters.
```

Comment:
```suggestion
Lint filenames for invalid or troublesome characters, such as pipes, colons, and brackets.
```

Reply:
I would also suggest this change for the `manifest.json` as well, since that's what shows up in the description in the Community Plugins list.